crates/wdl-engine/src/config.rs

```diff
@@ -671,6 +681,34 @@ impl ScatterConfig {
     }
 }
 
+/// Represents the supported call caching modes.
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum CallCachingMode {
```

Comment:
This brings up an interesting case, which is that I believe this is the first hint that we've implemented that wasn't outlined in the spec. The question arises as to what to do with this. I think we certainly do **not** want to just implement it and never try to get this into the spec. What do you think our policy should be on this kind of thing?